hatch_build.py

```diff
@@ -370,8 +370,7 @@
     # Required for python 3.9 to work with new annotations styles. Check package
     # description on PyPI for more details: https://pypi.org/project/eval-type-backport/
     'eval-type-backport>=0.2.0;python_version<"3.10"',
-    # temporarily pinning fastapi after the recent release started breaking tests
-    "fastapi[standard]>=0.112.2,!=0.115.10",
```

Comment:
I do not think we should do it. They have not yanked 0.115.10 and that exclusion is actually correct -  we will still get 0.115.11 and we will skip 0.115.10 - which for us is broken.

Reply:
Thanks. I just updated the PR to add a comment as to why we skip 0.115.10